app/views/forms/_form.json.jbuilder

```diff
@@ -1,5 +1,5 @@
 json.extract! form, :id, :name, :description, :created_at, :updated_at, :questions, \
```

Comment:
Do we want to add published_by to the serializers too, so it shows up in API calls?